tests/test_data_portal.py

```diff
@@ -406,7 +406,7 @@ def test_get_adjustments(self, data_frequency, field):
         day = calendar.day
         dividend_date = self.trading_days[2]
 
-        prev_day_price = 1.005
+        prev_day_price = 1.006
```

Comment:
is this also not perfectly representable?

Reply:
This value is fine, but we are trying to write 1.0055, which ends up being 1006 after scaling and rounding.

Reply:
For 1.0055 to 1.006, cannot change the OHLC_RATIO ？I changed it to 100000 for 5 decimal FX csv data.